eclair-core/src/main/scala/fr/acinq/eclair/channel/Channel.scala

```diff
@@ -222,8 +222,13 @@ class Channel(val nodeParams: NodeParams, val wallet: EclairWallet, remoteNodeId
             case None =>
               // in all other cases we need to be ready for any type of closing
               // TODO: should we wait for an acknowledgment from the watcher?
-              blockchain ! WatchSpent(self, data.commitments.commitInput.outPoint.txid, data.commitments.commitInput.outPoint.index.toInt, data.commitments.commitInput.txOut.publicKeyScript, BITCOIN_FUNDING_SPENT)
-              blockchain ! WatchLost(self, data.commitments.commitInput.outPoint.txid, nodeParams.minDepthBlocks, BITCOIN_FUNDING_LOST)
+              // avoid herd effect at startup because watch-spent are intensive in terms of rpc calls to bitcoind
```

Comment:
Maybe we should start with only delaying channels that are going to `NORMAL`, not the `CLOSING` ones?

Reply:
Good idea

Reply:
I've even restricted to the router watches. We'll see how it performs, given that the expensive path is to recursively look for spending transactions block-by-block, which we only do for channels.